rskj-core/src/main/java/co/rsk/peg/union/UnionResponseCode.java

```diff
@@ -0,0 +1,26 @@
+package co.rsk.peg.union;
+
+public enum UnionResponseCode {
+    SUCCESS(0),
+    UNAUTHORIZED_CALLER(-1),
+    // Invalid value when The amount requested, combined with previously requested amounts, exceeds the current locking cap value.
+    // or invalid value when increasing union bridge locking cap
```

Comment:
```suggestion
    // New cap value when increasing locking cap is invalid.
    // The rBTC requested, combined with previously requested rBTC, surpasses locking cap.
    // The rBTC returned from Union exceeds the one sent from the Bridge.
```

something like that? To describe in detail the three possible scenarios for this response value

Reply:
Updated